### ISSUE TITLE
betaC calculation should richness be extrapolated?

### ISSUE THREAD

**Issue by dmcglinn** [MoBiodiv/mobr#263]
Hey @T-Engel I'm testing out the recent merge of the beta_C code and I notice I was getting the following warning: 

```
beta_C(inv_comm[1:2, ], 0.9787)
    142 
1.46706 
attr(,"C")
[1] 0.9787
attr(,"N")
[1] 142
Warning messages:
1: In FUN(newX[, i], ...) :
  "effort" larger than total number of individuals returning S
2: In FUN(newX[, i], ...) :
  "effort" larger than total number of individuals returning S
```
Those warning messages are issued by the function rarefaction when the effort to compute S for is greater than the observed S of the sample. 

Should we add the argument `extrapolate = TRUE` to the call of the rarefaction function here: 
https://github.com/MoBiodiv/mobr/blob/dev/R/beta_C.R#L159-L161

Thanks!

**Comment by T-Engel** [comment]
Oh yes. When I copied the code into mobr I changed it to use our rarefaction() function rather than the D0() I copied from inext. I forgot that rarefaction() does not extrapolate by default. I'll change it

**Comment by dmcglinn** [comment]
Let me change it if that’s ok. I’m making other changes right now

Dan

**Comment by T-Engel** [comment]
Alright. Thanks.

#################################################################
Thore Engel (“Tory”)
Doctoral researcher / Doktorand
Biodiversity synthesis
German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig
Martin Luther University Halle-Wittenberg
Email     ***@***.******@***.***>

Pronouns / Pronomen: he / er

iDiv is a research centre of the DFG – Deutsche Forschungsgemeinschaft
#################################################################

Von: Dan McGlinn ***@***.***>
Gesendet: Donnerstag, 24. Juni 2021 17:02
An: MoBiodiv/mobr ***@***.***>
Cc: Engel, Thore ***@***.***>; Assign ***@***.***>
Betreff: [Extern] Re: [MoBiodiv/mobr] betaC calculation should richness be extrapolated? (#263)

Let me change it if that’s ok. I’m making other changes right now

Dan

**Comment by dmcglinn** [comment]
This was fixed in 1090e397de
